catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraCreatorsTest.scala

```diff
@@ -83,9 +84,151 @@ class SierraCreatorsTest extends FunSpec with Matchers {
 
     val creators = transformer.getCreators(bibData)
 
-    creators should contain only Person(
-      label = name,
-      prefix = Some(prefixes.mkString(" ")))
+    creators should contain only Unidentifiable(
+      Person(label = name, prefix = Some(prefixes.mkString(" "))))
+  }
+
+  it("extracts the creator identifier from marcTag 100 0") {
+    val name = "Carrot Ironfoundersson"
+    val code = "123456"
+
+    val bibData = SierraBibData(
+      id = "1234567",
+      title = None,
+      varFields = List(
+        VarField(
+          fieldTag = "p",
+          marcTag = "100",
+          indicator1 = "",
+          indicator2 = "",
+          subfields = List(
+            MarcSubfield(tag = "a", content = name),
+            MarcSubfield(tag = "0", content = code))))
+    )
+
+    val creators = transformer.getCreators(bibData)
+
+    val sourceIdentifier = SourceIdentifier(
+      IdentifierSchemes.libraryOfCongressNames,
+      "Person",
+      code)
+    creators should contain only Identifiable(
+      Person(label = name),
+      sourceIdentifier = sourceIdentifier,
+      identifiers = List(sourceIdentifier))
+  }
+
+  it(
+    "extracts the creator identifier removing trailing and leading from marcTag 100 0") {
+    val name = "Carrot Ironfoundersson"
+    val code = " 123456 "
+
+    val bibData = SierraBibData(
+      id = "1234567",
+      title = None,
+      varFields = List(
+        VarField(
+          fieldTag = "p",
+          marcTag = "100",
+          indicator1 = "",
+          indicator2 = "",
+          subfields = List(
+            MarcSubfield(tag = "a", content = name),
+            MarcSubfield(tag = "0", content = code))))
+    )
+
+    val creators = transformer.getCreators(bibData)
+
+    val sourceIdentifier = SourceIdentifier(
+      IdentifierSchemes.libraryOfCongressNames,
+      "Person",
+      code.trim)
+    creators should contain only Identifiable(
+      Person(label = name),
+      sourceIdentifier = sourceIdentifier,
+      List(sourceIdentifier))
+  }
+
+//  it("extracts the creator identifier removing trailing and leading from marcTag 100 0") {
```

Comment:
We should comment on why this test is commented out. I would also suggest we quarantine it rather than comment it out using the [ignore tag](http://www.scalatest.org/user_guide/tagging_your_tests).

Reply:
oops 👍